src/styles.css

```
@@ -1,311 +1,487 @@
-  font-size: 16px;
-  image-rendering: pixelated;
-  image-rendering: -moz-crisp-edges;
-  image-rendering: crisp-edges;
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.3.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)